documentation/docs/developers/connectors/task3.md

```diff
@@ -0,0 +1,193 @@
+
+### Cancelling Orders
+
+The `execute_cancel` function is the primary function used to cancel any particular tracked order. Below is a quick overview of the `execute_cancel` function
+
+Function<div style="width:150px"/> | Input Parameter(s) | Expected Output(s) | Description
+---|---|---|---
+`execute_cancel` | symbol: `str`,<br/>order_id: `str` | order_id: `str` | Function that makes API request to cancel an active order and returns the `order_id` if it has been successfully cancelled or no longer needs to be tracked.<br/><table><tbody><tr><td bgcolor="#ecf3ff">**Note**: This function also stops tracking the order by calling the `c_stop_tracking_order` and `c_trigger_event` functions. </td></tr></tbody></table>
+   
+### Tracking Orders & Balances
+
+The `Market` class tracks orders in several ways:
+
+- Listening on user stream<br/>
+This is primarily done in the `_user_stream_event_listener` function. This is only done when the exchange has a WebSocket API.
+
+- Periodic status polling<br/>
+This serves as a fallback for when user stream messages are not caught by the `UserStreamTracker`. This is done by the `_status_polling_loop` 
+
+The table below details the **required** functions to implement
+
+Function<div style="width:150px"/> | Description
+---|---
+`_user_stream_event_listener`| Update order statuses and/or account balances from incoming messages from the user stream.
+`_update_balances`| Pulls the REST API for the latest account balances and updates `_account_balances` and `_account_available_balances`.
+`_update_order_status`| Pulls the REST API for the latest order statuses and updates the order statuses of locally tracked orders.
+
+If the exchange doesn't provide user balance updates in real-time (web socket), you will need to set `self._real_time_balance_update = False` in the market constructor (init). 
+ 
+And, you will need to take `in_flight_orders` snapshot during `_update_balances` as below:
+
+```python
+    self._in_flight_orders_snapshot = {k: copy.copy(v) for k, v in self._in_flight_orders.items()}
+    self._in_flight_orders_snapshot_timestamp = self._current_timestamp
+``` 
+
+This is so that the connector can use default current balance calculation (in `market_base`) for available balances.
+ 
+!!! tip
+    Refer to [Order Lifecycle](/developers/connectors/order-lifecycle) for a more detailed description on how orders are being tracked in Hummingbot.
```

Comment:
This link is working but is missing from the "nav" config in `mkdocs.yml` file. See warning below.

![image](https://user-images.githubusercontent.com/50150287/91408986-57cc9900-e877-11ea-8e70-b9e041cc0a35.png)

Reply:
noted, have merged the topic into tutorial, will delete the topic.